Codecademy/gamut
# feat(Tooltip): add aria-described by to floating tooltip

### Overview


Swaps `aria-label` to `aria-describedby` for our floating ToolTip


### PR Checklist

- [x] Related to JIRA ticket: [web-2424]

#### Testing Instructions



* Go to the Tooltip Story -> Placement section
* Turn on VO 
* Tab to Floating Section
* Should hear tooltips read when tabbing to buttons + info icons

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/36449)  | <= |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/3462)  | <=   |
